frontend/public/learn/Interactive-UI-Elements.elm

```diff
@@ -46,38 +51,40 @@ and then dive into how it works.
 We are going to make three synced checkboxes. Changing one of them will change
 the two others:
 
-|]
+"""
 
-rest = [markdown|
+rest : Element
+rest = Markdown.toElement """
 
 Here is the code we need to make that happen. Don't worry about the details too
 much yet. This is more to get a feel for the API so we know what we are working
 towards when we dive into the details:
 
 ```haskell
```

Comment:
I think we can use `elm` instead of `haskell` here. I checked and we're adding to the parser GitHub uses, and I think it's the same one for the markdown package.